docs/UserGuide.md

```diff
@@ -102,87 +92,147 @@ There are three main areas in TutorBuddy:
+  e.g. `find_student KEYWORD [MORE_KEYWORDS]` can be used as `find_student John Samuel` or as `find_student John`.
 
 * Parameters can be in any order.<br>
   e.g. if the command specifies `n/NAME p/PHONE_NUMBER`, `p/PHONE_NUMBER n/NAME` is also acceptable.
 
 * If a parameter is expected only once in the command but you specified it multiple times, only the last occurrence of the parameter will be taken.<br>
-  e.g. if you specify `p/12341234 p/56785678`, only `p/56785678` will be taken.
+  e.g. if you specify `p/98765432 p/99999999`, only `p/98765432` will be taken.
 
-* Extraneous parameters for commands that do not take in parameters (such as `emails`, `help`, `exit` and `clear`) will be ignored.<br>
+* Extraneous parameters for commands that do not take in parameters (such as `emails`, `help`, `exit`, `clear` and `list_student`) will be ignored.<br>
   e.g. if the command specifies `help 123`, it will be interpreted as `help`.
 
+* Indexes **must be positive whole numbers** 1, 2, 3, …
+
+* The `STUDENT_INDEX` refers to the index number shown in the [displayed student list](#application-overview).
+
+* The `SESSION_INDEX` refers to the index number shown in the [displayed session list](#application-overview).
+
+* Time must be input in the format, `hh:mm`.
+  e.g. `13:00` is allowed, but not `1300`.
+  
+* Date must be input in the format, `yyyy-mm-dd`.
+  e.g. `2021-01-01` is allowed, but not `01-01-2021`.
+  
+  
 </div>
 
-### Managing Students Commands
+The following subsections will elaborate on the specific details of each TutorBuddy's command.
+
+### General
+
+#### Viewing help : `help`
 
-#### Listing all students: `list_student`
+Shows a message explaining how to access the help page.
 
-Shows a list of all students in the TutorBuddy
+![help message](images/helpMessage.png)
+
+Format: `help`
+
+#### Listing all students and sessions: `list_student`
+
+Displays list of all students and all sessions in TutorBuddy.
+
+![list](images/command_list.png)
 
 Format: `list_student` <br>
 
-#### Locating student profile by name: `find_student`
+#### Clearing all data : `clear`
+
+Clears all student and session data.
+
+<div markdown="span" class="alert alert-primary">:bulb: Tip:
+TutorBuddy provides sample data to allow you to try out the application easily. <br>
+Run the `clear` command to start working with TutorBuddy in a clean slate.
+</div>
+
+Format: `clear`
+
+#### Exit the program: `exit`
+
+Exits the program
+
+Format: `exit`
+
+### Managing Students
+
+
+#### Adding a student: `add_student`
+
+Adds a student to TutorBuddy.
+
+Format: `add_student n/NAME p/STUDENT_PHONE_NUMBER e/EMAIL a/ADDRESS l/STUDY_LEVEL g/GUARDIAN_PHONE_NUMBER r/RELATIONSHIP_WITH_GUARDIAN` <br>
+
+<div markdown="block" class="alert alert-info">
+:information_source: `STUDENT_PHONE_NUMBER`, `GUARDIAN_PHONE_NUMBER` should be in Singapore's phone formatting (i.e. starting with either 6, 8 or 9 and with 8 digits)
```

Comment:
Not sure if it's just on my end but i notice the `STUDENT_PHONE_NUMBER` and `GUARDIAN_PHONE_NUMBER` not showing up as intended inside of the block.

Reply:
Same soln as above, worth a try. @JonahhGohh